### DISCUSSION TITLE
cannot find any crud operations on milvus 2.0.0rc1

### DISCUSSION THREAD

**Discussion by sadakmed** [milvus-io/milvus#6394] [Q&A and General discussion]
I'm interested in deleting and updating embeddings, I use milvus 2.0.0rc1  coz I found that I can have a collection that contains the embeddings and another field -id, type string-. the documentation is so poor, on version 2.0.0. 
any help on the operations, or indications for the documents.

another questions, is what's the difference between pymilvus and  pymiluvs_orm?

**Comment by xiaofan-luan** [comment]
hi sadakmed, delete is still under implementation for 2.0.0-rc1. The expected date to release is end of August, if you want to use it right now maybe you can start from 1.1.x.  

Also string type id is under implementation.

pymiluvs_orm is the recommended api for milvus 2.0, object relational mapping is a coding style helps people to ignore the implementation details and focus on business logic. We will implement a embededded libray under pymilvus_orm.

Please help us on improve 2.0.0 document, you can directly send pull request or open issues if you feel anything is bad.

**Reply by sadakmed** [reply]
is string type id still under implementation in 1.1.1?

**Reply by xiaofan-luan** [reply]
nope, string is not supported in either 1.x or 2.x. it's still under implementation at 2.x. delete is impleted in 1.x
